KStudy.js

```diff
@@ -131,11 +133,23 @@ function scrape(doc, url) {
 		var containerParts = containerValue.match(/(.*)\s+(\d+)\D\s*(\d+)/);
 		if (containerParts) {
 			item.publicationTitle = containerParts[1];
-			item.volume = containerParts[2];
+			if (/\d{4}/.test(containerParts[2])) {
+				containerParts[2] = null;
```

Comment:
Add a comment why you are doing this. Moreover, I guess you don't need to delete the value in the first case, but rather simply want to concentrate on the second case. Is this correct? Then you could instead try something like
```js
if (containerParts[2].length<4) { // or the regexp if really needed
   item.volume = containerParts[2];
}
```

Reply:
Oh, okay thank you for pointing that out.
The reason I added that line was that I found that sometimes the pages have published year in place of the volume number. So sometimes it would appear as Vol. 2013, and I didn't want that to happen, but your correction makes much more sense. Thank you. I'll fix that.